### ISSUE TITLE
SBGN brick: Multimerization

### ISSUE THREAD

**Issue by ugurdogrusoz** [iVis-at-Bilkent/newt#667]
As described in the related document and also shown [here](http://sbgnbricks.org/BKO/full/entry/PD/BKO:0000221/), let's implement multimerization. Notice that, unlike previous ones, we should let the user choose the type of regulator (stimulation or inhibition).

**Comment by msalihaltun** [comment]
https://github.com/iVis-at-Bilkent/newt/commit/9ea384db1055207efde029793a4ce135f0c78a91 implements multimerization.

**Comment by ugurdogrusoz** [comment]
Some improvements:
- Put the type of regulation *after* multimer checkbox and count (leave some space in between) as regulator name and type belong to the regulating node, whereas regulation type is related to the edge from the regulator node to the process.
- When multimer cardinality is below 2, we should not display any cardinality on the input edge at all.
- When orientation is top-to-bottom, multimer cardinality is too close to the input macromolecule and blocked by it.

**Comment by msalihaltun** [comment]
I think the edge cardinality when vertical is about how we render the cardinality inside `sbgnviz.js`. I can see the same problem in the daily deployment.

![image](https://user-images.githubusercontent.com/26147738/119322598-6faa3f80-bc86-11eb-9888-eba1b6c3c2dd.png)

**Comment by ugurdogrusoz** [comment]
OK. If the location of the cardinality is determined automatically, this is a separate issue. Please create a new issue for this to fix later.

**Comment by ugurdogrusoz** [comment]
1. I don't know if this is intentional but I think all can go in a single row:
![Screenshot 2021-05-24 135410](https://user-images.githubusercontent.com/3874988/119337785-e56ad700-bc97-11eb-9941-908e8e20d857.png)

**Comment by msalihaltun** [comment]
That's strange. It all goes in the same row for me and pretty much everything is fixed length. Is it the same for association brick as well?
